html/partials/compare/repositories.html.haml

```diff
@@ -18,6 +18,11 @@
             - if @diff["repositories"].only_in2
               %span.summary-part
                 #{@description_b.name}: #{@diff["repositories"].only_in2.length} repositories
+            - if @diff["repositories"].changed
+              %span.summary-part
+                %a{ href: "#repositories_changed" }
+                  changed
+                = ": #{@diff["repositories"].changed.length} repositories"
```

Comment:
You're right, but I'd propose to tackle the pluralization in a separate pull request. That  one is quite big already and there are many other places also in the current code where this needs to be fixed.

Fine with you?